### PR TITLE
SEO-190701--EJ2-ASP.NET-CORE-pivot-table-how-to-bind-complex-data-to-…

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | SEMrush and Bing reports- Description too long|
|Content Task Link/Mail Screenshot | NA|
|Hotfix| https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/3444|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/190701|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B4631D723-2EC0-4999-985A-2FDC8DBD30C5%7D&file=Faith.xlsx&action=default&mobileredirect=true|

Changes Made | Reason for change |
|-- | -- |
|Changed- Description too long | To avoid CI issues since in SEO description should range from 100 to 170 character count |

### CODE REVIEW

**Comment by Faith-Atieno** [comment]
rebuild

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Convert complex JSON to flat JSON and assign it to the pivot table in ##Platform_Name## Pivot Table Component
-description: Learn here all about how to convert complex JSON to flat JSON and assign it to the pivot table in Syncfusion ##Platform_Name## Pivot Table component of syncfusion and more.
+title: Convert complex JSON to flat JSON and assign pivot table | Syncfusion
+description: Learn here all about how to convert complex JSON to flat JSON and assign it to the pivot table in ##Platform_Name## component of syncfusion and more.
```
Comment (Yvone-Atieno) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md:4]: Update proper title and H1

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md (hunk 1 above)
Comment (Faith-Atieno) [reply ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md:4]: resolved

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 platform: ej2-asp-core-mvc
 control: Convert complex JSON to flat JSON and assign it to the pivot table 
 documentation: ug
 publishingplatform: ##Platform_Name## 
 ---
 
 <!-- markdownlint-disable MD009 -->
-# Convert complex JSON to flat JSON and assign it to the pivot table
+# Bind complex data to the pivot table in ##Platform_Name##
```
Comment (Yvone-Atieno) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md:12]: Update proper H1

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md (hunk 2 above)
Comment (Faith-Atieno) [reply ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md:12]: resolved

File: ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Convert complex JSON to flat JSON and assign it to the pivot table in ##Platform_Name## Pivot Table Component
-description: Learn here all about how to convert complex JSON to flat JSON and assign it to the pivot table in Syncfusion ##Platform_Name## Pivot Table component of syncfusion and more.
+title: Convert complex JSON to flat JSON in ##Platform_Name## pivot table | Syncfusion
```
Comment (Yvone-Atieno) [inline ej2-asp-core-mvc/pivot-table/EJ2_ASP.NETCORE/how-to/bind-complex-data-to-the-pivot-table.md:3]: Check the updated title

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Pull request closed without merging**